nodl_python/nodl/errors.py

```diff
@@ -15,13 +15,37 @@
 from lxml import etree
 
 if TYPE_CHECKING:  # pragma: no cover
+    from .types import Node
     import rclpy.qos
```

Comment:
Is this a performance optimization? If so, you can probably stop using `TYPE_CHECKING` unless it's to prevent circular imports. I can't imagine there's a measurable performance hit here.

Reply:
Fixed in 967d461